lint-python/action.yml

```diff
@@ -14,17 +18,33 @@ runs:
 
     - uses: moneymeets/moneymeets-composite-actions/check-git-diff@master
 
+    - run: echo "ruff-available=$(poetry show ruff -q ; echo $?)" >> $GITHUB_OUTPUT
+      id: detect-ruff
+      shell: bash
+
+    - name: Run linter
+      if: steps.detect-ruff.outputs.ruff-available == 0
```

Comment:
Just wanted to say that it must be exactly the other way around, but you changed it already:

```
(moneymeets-backoffice-py3.11) zaytsev@parallels:~/src/moneymeets-backoffice$ poetry show ruff -q ; echo $?
0

(fondsnet-sdk-py3.11) zaytsev@parallels:~/src/fondsnet-sdk$ poetry show ruff -q ; echo $?
1
```

Reply:
Yeah, already updated and tested. :)